### PR TITLE
adding prereqs to readme

### PR DESCRIPTION
**Pull request by cmastr**
adding some prerequisites as I figure out which ones are needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,25 @@ This repository contains the documentation website code and source files for htt
...
 - A list of READMEs
 - Code of conduct
 - License agreement
 
+## Prerequisites 
+The following are a list of prerequisites you may want to consider downloading and installing to successfully contribute to the Segment docs repo:
+
+1. Download and install a source code editor like [Visual Studio Code](https://code.visualstudio.com/download). 
+1. Download and install a package manager such as [Homebrew](https://brew.sh/) to install new software.
+1. [Create an account on GitHub](https://docs.github.com/en/get-started/quickstart/creating-an-account-on-github) if you don't have one already, or sign in with your existing Github account.
```
Comment (stayseesong) [inline README.md:25]:
```suggestion
1. [Create an account on GitHub](https://docs.github.com/en/get-started/quickstart/creating-an-account-on-github) if you don't have one already, or sign in with your existing GitHub account.
```

File: README.md [hunk 2]
```diff
@@ -11,11 +11,25 @@ This repository contains the documentation website code and source files for htt
...
+1. Add the GitHub extension to your editor (in VSC: https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github)
```
Comment (stayseesong) [inline README.md:26]:
```suggestion
1. Add the GitHub extension to your editor (in VSC: https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github).
```

File: README.md [hunk 3]
```diff
@@ -11,11 +11,25 @@ This repository contains the documentation website code and source files for htt
...
+1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the segment docs repo.
```
Comment (stayseesong) [inline README.md:27]:
```suggestion
1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the Segment docs repo.
```

File: README.md [hunk 4]
```diff
@@ -11,11 +11,25 @@ This repository contains the documentation website code and source files for htt
...
+1. Download and install the latest version of [node.js](https://nodejs.org/en/download). Consider using a version manager such as [ASDF](https://github.com/asdf-vm/asdf) or [nvm-windows](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows)
```
Comment (stayseesong) [inline README.md:28]:
```suggestion
1. Download and install the latest version of [node.js](https://nodejs.org/en/download). Consider using a version manager such as [ASDF](https://github.com/asdf-vm/asdf) or [nvm-windows](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows).
```

File: README.md [hunk 5]
```diff
@@ -11,11 +11,25 @@ This repository contains the documentation website code and source files for htt
...
+1. Install dependencies 
+  `npm i`
+1. Install [vale](https://docs.errata.ai/vale/install) (an [ASDF plugin](https://github.com/osg/asdf-vale) is also available).
```
Comment (stayseesong) [inline README.md:31]:
```suggestion
1. Install [vale](https://docs.errata.ai/vale/install). An [ASDF plugin](https://github.com/osg/asdf-vale) is also available.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@cmastr rather than marking this with a "fixed-site" label, I feel like it should be labeled as "back-end". wdyt?

**Comment by cmastr** [comment]
> @cmastr rather than marking this with a "fixed-site" label, I feel like it should be labeled as "back-end". wdyt?

This sounds good! Changed label. And added all of your comments.

**Review by stayseesong — APPROVED** [review]

**Pull request merged**